plugins/autocertkit

```diff
@@ -609,6 +609,63 @@ def deploy_iperf(session, args):
     return json_dumps("OK")
 
 
+@log_exceptions
+def deploy_vf_driver(session, args):
+    """Install VF driver on a specified VM"""
+    log.debug("deploy VF driver plugin call")
+    vm_ref = validate_exists(args, 'vm_ref')
+    username = validate_exists(args, 'username')
+    password = validate_exists(args, 'password')
+    package  = args['package']
+    driver_name = args['driver_name']
+
+    if not session.xenapi.VM.get_is_control_domain(vm_ref):
+        i = 0
+        limit = 10
+        while i < limit:
+            try:
+                install_ssh_key(session, vm_ref, username, password)
+                install_vf_driver(session, vm_ref, username, password, package, driver_name)
```

Comment:
So presumably at the moment we are limited to VMs which support RPM packages right?

Reply:
yes, otherwise we don't how to install a arbitrary VF driver pkg, and rpm is centos standard package format to support.